pyanaconda/ui/gui/spokes/system_purpose.py

```diff
@@ -0,0 +1,265 @@
+# system purpose spoke class
+#
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+import gi
+gi.require_version("Gtk", "3.0")
+gi.require_version("Pango", "1.0")
+
+from gi.repository import Gtk, Pango
+
+from pyanaconda.flags import flags
+from pyanaconda.core.i18n import _, CN_
+from pyanaconda.core import constants
+from pyanaconda.modules.common.constants.services import SUBSCRIPTION
+
+from pyanaconda.ui.gui.spokes import NormalSpoke
+from pyanaconda.ui.gui.utils import escape_markup
+from pyanaconda.ui.categories.system import SystemCategory
+from pyanaconda.ui.communication import hubQ
+
+from pyanaconda.anaconda_loggers import get_module_logger
+log = get_module_logger(__name__)
+
+__all__ = ["SystemPurposeSpoke"]
+
+
+class SystemPurposeSpoke(NormalSpoke):
```

Comment:
Please, attach a screenshot with the new spoke.

Reply:
Good point, will do that.